AmazonAdvertisingApi/Versions.php

```diff
@@ -5,6 +5,6 @@ class Versions
 {
     public $versionStrings = array(
         "apiVersion"         => "v2",
-        "applicationVersion" => "1.2"
+        "applicationVersion" => "2.0"
     );
 }
```

Comment:
Not sure if we're just getting it working or tidying things up for the future, I'd not bother changing this. 

Having a getter for an array that contains the version values seems more convoluted then having a getter function for each one so you don't need to remember what the keys are? Or better constants so you don't need an instance of the class and save another line.

The only place it seems to have been used originally is in the __construct for client.php

Reply:
Not sure I am following. 
i) Changing the version because we are making breaking changes, so I want it to be as obvious as possible for if people are just bringing in the latest version in their composers. I feel like that isn't the point you were making though. 

ii) Agree that seems quite convoluted. It is the same as the way the tool seems to be structured for other things. Happy to change to getters for individual values if you think it is worth it?